### DISCUSSION TITLE
Request: ENV var to enable web search toggle by default for every user

### DISCUSSION THREAD

**Discussion by dillfrescott** [open-webui/open-webui#5393] [Ideas]
This would be nice because I am often having to toggle it on for every chat in new tabs or if i refresh the page. Having an env variable to just default it to on would be super nice!

**Comment by dillfrescott** [comment]
Some sort of "smart mode" would also be nice where the model can decide whether or not it should even show the search dialog or attempt a search.

I noticed that if the toggle is on the model wont search things that dont need to be, which is good, but the searching dialog still comes up regardless.

**Comment by ghost** [comment]
Smart mode can be achieved by using a search tool from the website along with an auto-tool filter... although I have no idea if auto-tool filters are working at the moment (they kept breaking on me). What I would love, similar to this, is a toggle on a model to have function toolkits turned on by default.

**Comment by tjbck** [comment]
Perhaps using `/?web-search=true` url param cover your use case?

**Reply by dillfrescott** [reply]
I did not know about this. this may work. Thank you for this piece of information.
